src/careamics/config/vae_algorithm_model.py

```diff
@@ -69,6 +69,7 @@ class VAEAlgorithmConfig(BaseModel):
     loss: Literal["musplit", "denoisplit"]
     model: Union[LVAEModel, CustomModel] = Field(discriminator="architecture")
 
+    # TODO different channels for nm should be where ?
```

Comment:
In #205 we decided to adopt a new configuration for Noise Models. Specifically:
- for each output channel (e.g., 1 in HDN, >1 in denoiSplit) we define an actual noise model (for the moment the only choice is `GaussianMixtureNoiseModel`, but in the future we will also have an `HistogramNoiseModel`).
- the previous `DisentNoiseModel`, which gathers the noise models for the single channels, has been replaced by a more clear `MultiChannelNoiseModel`, which basically gathers the different `GaussianMixtureNoiseModel`'s in a list (or something logically similar to a list). 
- therefore, in this new configuration, whenever we need the noise model (e.g., in likelihood/loss computations), we access it through the `MultiChannelNoiseModel` as it provides all the functionalities to compute log-likelihood and related stuff. In addition, this new structure hides the internal parameters of the noise models, which are set a-priori (through training or loading from `path`) and should not be touched during LVAE training or evaluation.

Reply:
I guess this is what is happening. Sorry if I'm missing something